site/src/App/DocNavigation/DocNavigation.tsx

```diff
@@ -206,7 +206,7 @@ export const DocNavigation = () => {
         <Heading level="2" component="h1">
           {docsName}
         </Heading>
-        <DocNavigationBar title="Subnavigation">
+        <DocNavigationBar title="Sidenavigation">
```

Comment:
Is this a result of find and replace? If its a deliberate change it should have a space